avm/res/cdn/profile/tests/e2e/afd/main.test.bicep

```diff
@@ -0,0 +1,121 @@
+targetScope = 'subscription'
+
+metadata name = 'As Azure Front Door'
+metadata description = 'This instance deploys the module as Azure Front Door.'
+
+// ========== //
+// Parameters //
+// ========== //
+
+@description('Optional. The name of the resource group to deploy for testing purposes.')
+@maxLength(90)
+param resourceGroupName string = 'dep-${namePrefix}-cdn.profiles-${serviceShort}-rg'
+
+@description('Optional. The location to deploy resources to.')
+param resourceLocation string = deployment().location
+
+@description('Optional. A short identifier for the kind of deployment. Should be kept short to not run into resource-name length-constraints.')
+param serviceShort string = 'cdnpafd'
+
+@description('Optional. A token to inject into the name of each resource.')
+param namePrefix string = '#_namePrefix_#'
+
+// ============ //
+// Dependencies //
+// ============ //
+
+// General resources
+// =================
+resource resourceGroup 'Microsoft.Resources/resourceGroups@2021-04-01' = {
+  name: resourceGroupName
+  location: resourceLocation
+}
+
+module nestedDependencies 'dependencies.bicep' = {
+  scope: resourceGroup
+  name: '${uniqueString(deployment().name, resourceLocation)}-nestedDependencies'
+  params: {
+    storageAccountName: 'dep${namePrefix}cdnstore${serviceShort}'
+    managedIdentityName: 'dep-${namePrefix}-msi-${serviceShort}'
+    location: resourceLocation
+  }
+}
```

Comment:
It appears non of the dependencies are used?

Reply:
good catch, thanks. removed: https://github.com/Azure/bicep-registry-modules/pull/1106/commits/8fe70c4ad3099cbb6a612875d190650e85134319